### PR TITLE
Adding QEmu/KVM download link.

### PR DESCRIPTION
**Pull request by geertj**

### CODE REVIEW

**Comment by geertj** [comment]
@Fryguy @chessbyte  @matthewd could one of you review and commit before Wed COB? I am referring to a QEmu download option in the LWN article.

File: data/deploy_types.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+qemu:
+  name: QEmu/KVM
+  download_platform: openstack
+  ext: qc2
```
Comment (Fryguy) [inline data/deploy_types.yml]: Can you move this down so that this file is sorted by name (i.e. between rhev and vsphere)?

Comment (geertj) [inline data/deploy_types.yml]: You mean between ovirt and RHEV, right?  :)

Comment (Fryguy) [inline data/deploy_types.yml]: Yes, I can't alphabet :P

**Comment by Fryguy** [comment]
:+1: will merge when green.

**Pull request merged**